src/components/common/Button.tsx

```diff
@@ -1,24 +1,24 @@
-import { ButtonHTMLAttributes } from 'react';
+import { ButtonHTMLAttributes } from "react";
 
 interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
-  onClick: () => void
-  text: string
-  className?: string
+  onClick: () => void;
+  text: string;
+  className?: string;
   children?: React.ReactNode;
-  type?: 'button' | 'submit' | 'reset'
-  size?: 'small' | 'medium' | 'large';
+  type?: "button" | "submit" | "reset";
+  size?: "small" | "medium" | "large";
 }
 
-
 const Button: React.FC<ButtonProps> = ({ onClick, text, children }) => {
```

Comment:
```suggestion
const Button = ({ onClick, text, children }) => {
```
I think as we discussed before you don't need React.FC types